aiidalab_widgets_base/utils/loaders.py

```diff
@@ -0,0 +1,37 @@
+from importlib.resources import Package, files
+
+from IPython.display import Javascript, display
+
+
+def load_css_stylesheet(package: Package, filename: str = ""):
```

Comment:
I suggest changing `package` to `module`, as it accepts a module.

Reply:
I think the name is correct, it accepts a package (i.e. a folder), not a module (which would be an individual python file).

Reply:
`package` is more like a Python pacage, like the `aiidalab_widgets_base`. From the example, 

```python
    from .static import styles
    from .utils.loaders import load_css_stylesheet

    load_css_stylesheet(package=styles)
```
It passes the module `styles` into it.  'aiidalab_widgets_base.static.styles' is a module, not a package.

Reply:
I don't think that's correct, citing e.g. from [this glossary](https://the-hitchhikers-guide-to-packaging.readthedocs.io/en/latest/glossary.html)

> **Module**
>     A python source code file (ex. mymodule.py), most often found in a package (mypackage/mymodule.py). You can import a module: import mymodule, or import mypackage.mymodule if it lives in mypackage.
>
> **Package** 
>     A directory containing an __init__.py file (ex. mypackage/__init__.py), and also usually containing modules (possibly along with other packages). You can import a package: import mypackage
>     A package should not be confused with a compressed archive file used to install Python code.

`aiidalab_widgets_base` is a package, containing `static` subpackage which itself contains `styles` subpackage. After all, the type of this variable *is* called `Package`.

Reply:
Thanks for the clarification! But it is still very weird to call, e.g., `aiida.orm`, a package.